service-workers/service-worker/fetch-event-redirect.https.html

```diff
@@ -966,56 +842,5 @@
 }, 'Non-navigation, error redirect, same-origin mode Request redirected to ' +
    'cors with credentials should fail interception ' +
    'and response should not be redirected');
-
-promise_test(function(t) {
-  return redirect_fetch_test(t, {
-    name: 'nonav-error-nocors-redirects-to-sameorigin-creds',
-    redirect_dest: 'same-origin',
-    url_credentials: true,
-    request_init: {
-      redirect: 'error',
-      mode: 'no-cors'
-    },
-    // should reject because requests with 'error' RequestRedirect cannot be
-    // redirected.
-    should_reject: true
-  });
-}, 'Non-navigation, error redirect, no-cors mode Request redirected to ' +
-   'same-origin with credentials should fail interception ' +
-   'and response should not be redirected');
-
-promise_test(function(t) {
-  return redirect_fetch_test(t, {
-    name: 'nonav-error-nocors-redirects-to-nocors-creds',
-    redirect_dest: 'no-cors',
-    url_credentials: true,
-    request_init: {
-      redirect: 'error',
-      mode: 'no-cors'
-    },
-    // should reject because requests with 'error' RequestRedirect cannot be
-    // redirected.
-    should_reject: true
-  });
-}, 'Non-navigation, error redirect, no-cors mode Request redirected to ' +
-   'no-cors with credentials should fail interception ' +
-   'and response should not be redirected');
-
-promise_test(function(t) {
-  return redirect_fetch_test(t, {
-    name: 'nonav-error-nocors-redirects-to-cors-creds',
-    redirect_dest: 'cors',
-    url_credentials: true,
-    request_init: {
-      redirect: 'error',
-      mode: 'no-cors'
-    },
-    // should reject because requests with 'error' RequestRedirect cannot be
-    // redirected.
-    should_reject: true
-  });
-}, 'Non-navigation, error redirect, no-cors mode Request redirected to ' +
-   'cors with credentials should fail interception and response should not ' +
-   'be redirected');
```

Comment:
Same with these cases.  I'd prefer to keep coverage of all the possible inputs even if they are all hitting the same internal condition.  In the future the internal conditions may change, but the possible set of inputs is fixed.

Reply:
Yes I'm curious the reason to delete these too.